### PR TITLE
Added Spotless to build scripts and formatted Java code

### PR DESCRIPTION
**Pull request by rabidgremlin**

### CODE REVIEW

File: concord-server/server/src/main/java/com/rabidgremlin/concord/ConcordServerApplication.java [hunk 1]
```diff
@@ -118,7 +119,11 @@ private void setupJwtAuth(ConcordServerConfiguration configuration,
         .setRequireExpirationTime() // the JWT must have an expiration time
         .setRequireSubject() // the JWT must have a subject claim
         .setVerificationKey(new HmacKey(configuration.getJwtTokenSecret())) // verify the signature with the public key
-        .setJwsAlgorithmConstraints(new AlgorithmConstraints(ConstraintType.WHITELIST, org.jose4j.jws.AlgorithmIdentifiers.HMAC_SHA256)) // we only expect HMAC_SHA256 alg 
+        .setJwsAlgorithmConstraints(new AlgorithmConstraints(ConstraintType.WHITELIST, org.jose4j.jws.AlgorithmIdentifiers.HMAC_SHA256)) // we
+                                                                                                                                         // only
+                                                                                                                                         // expect
```
Comment (LaurenceTews) [inline concord-server/server/src/main/java/com/rabidgremlin/concord/ConcordServerApplication.java]: comment formatting has been bungled

File: concord-server/server/src/main/java/com/rabidgremlin/concord/dao/LabelsDao.java [hunk 2]
```diff
@@ -12,16 +12,16 @@
...
-	void deleteAllLabels();
-	
-	@SqlUpdate("REPLACE INTO labels(label, shortDescription, longDescription) VALUES (:label, :shortDescription,:longDescription)")
-    void upsert(@BindBean Label label);
+  @SqlUpdate("DELETE from labels")
+  void deleteAllLabels();
 
-	@SqlBatch("REPLACE INTO labels(label, shortDescription, longDescription) VALUES (:label, :shortDescription,:longDescription)")
-	void upsert(@BindBean List<Label> label);
+  @SqlUpdate("REPLACE INTO labels(label, shortDescription, longDescription) VALUES (:label, :shortDescription,:longDescription)")
```
Comment (LaurenceTews) [inline concord-server/server/src/main/java/com/rabidgremlin/concord/dao/LabelsDao.java]: Why is this changed to update?

**Comment by joedai** [comment]
The code was a bit mess without the format, much better now!
